src/Microsoft.Identity.Web/Resource/ScopesRequiredHttpContextExtensions.cs

```diff
@@ -19,34 +19,47 @@ public static class ScopesRequiredHttpContextExtensions
         /// <summary>
         /// When applied to an <see cref="HttpContext"/>, verifies that the user authenticated in the
         /// web API has any of the accepted scopes.
+        /// If there is no authenticated user, the reponse is a 401 (Unauthenticated).
         /// If the authenticated user does not have any of these <paramref name="acceptedScopes"/>, the
-        /// method throws an HTTP Unauthorized with the message telling which scopes are expected in the token.
+        /// method updates the HTTP response providing a status code Forbidden (403)
```

Comment:
```suggestion
        /// method updates the HTTP response providing a status code 403 (Forbidden)
```

Switching order to keep with convention of the others.